Client/Frontend/Browser/BrowserToolbar.swift

```diff
@@ -53,7 +75,14 @@ class BrowserToolbar: Toolbar {
         longPressGestureForwardButton = UILongPressGestureRecognizer(target: self, action: "SELdidLongPressForward:")
         forwardButton.addGestureRecognizer(longPressGestureForwardButton)
         forwardButton.addTarget(self, action: "SELdidClickForward", forControlEvents: UIControlEvents.TouchUpInside)
-        forwardButton.contentEdgeInsets = ButtonInset
+        forwardButton.contentEdgeInsets = NavButtonInset
+
+        stopReloadButton.setImage(ImageReload, forState: .Normal)
+        stopReloadButton.setImage(ImageReloadPressed, forState: .Highlighted)
+        stopReloadButton.accessibilityLabel = NSLocalizedString("Reload", comment: "Accessibility Label for the browser toolbar Reload button")
+        stopReloadButton.accessibilityHint = NSLocalizedString("Tap to stop or reload page", comment: "")
```

Comment:
If you change the a11y strings, it also needs to happen here. Because setting the loading property from `init()` has no effect. (The didSet will not be called)